io/lab: Fix typos (#308)
open-education-hub/operating-systems: content/chapters/io/lab/content/beyond-network-sockets.md

Now fill in the `TODO`s in `receive_unix_socket.c`.
Use the example at the bottom of [the UNIX socket man page](https://man7.org/linux/man-pages/man7/unix.7.html) if you get stuck at `bind()`.

> When you're done, compile it first and then run `send_unix_socket`.

Reviewer: ```suggestion
When you're done, compile the code and then run `send_unix_socket`.
```
